### ISSUE TITLE
Cars Assemble: Rework exercise so it only covers `numbers` and `arithmetic-operators`

### ISSUE THREAD

**Issue by junedev** [exercism/go#1936]
:warning: Behold! This issue should only be tackled after #1934 was completed.

## Motivation
Currently, the concept exercise `cars-assemble` teaches 4 concepts. We want to split that exercise into one exercise that teaches `numbers` and `arithmetic-operators` (`cars-assemble`) and one exercise that teaches `conditionals-if` and `comparison` (`vehicle-purchase`).

Why?
* This would help getting the concept tree in better shape. It makes more sense to have the basic types as the first "layer" below `basics` and then once that layer is completed you move on to the conditionals etc.
* 4 concepts for one exercise is really a lot even if they are not that difficult.
* Currently the exercise does not practice much of each concept, that could be improved with the split.

## Task
Change the `cars-assemble` concept exercise so it only teaches `numbers` and `arithmetic-operators` and adapt the exercise accordingly.

This involves the following.

- [ ] Remove `conditionals-if` and `comparison` from the concepts mentioned for this exercise in the root level `config.json` file.
- [ ] Remove the content on if statements and comparison from the exercise introduction.
- [ ] Change the tasks in the `instructions.md` file so that no if statements are needed anymore. For example, remove the `if` related tasks and add a new task that involves the `%` operator as this one is needed often in practice exercises. (Having just 3 tasks in total would be fine for now.) Keep in mind that at this point in the syllabus, the student only knows about the content in `basics` (exercise `lasagna`). That means all tasks should really just be about calculating some things.
- [ ] Update the blurb in the exercise `config.json` and add yourself as an author in case you added a completely new task.
- [ ] Update the `exemplar.go` file to match the new instructions.
- [ ] Update the tests in cars_assemble_test.go to match the new instructions.
- [ ] Update the `hints.md` file.
- [ ] Update the stub file (`cars_assemble.go`) to match the function signatures in the new exemplar.go file.
- [ ] Update the `design.md` file (and remove the stray comment about a C# port in there while you are at it).

## Docs

- [What are Concept Exercises?](https://exercism.org/docs/building/product/concept-exercises)
- [Concept Exercises](https://exercism.org/docs/building/tracks/concept-exercises)
- [Concepts](https://exercism.org/docs/building/tracks/concepts)

**Comment by junedev** [comment]
Note for maintainers: Let's not set `x:` labels for this issue until it can actually be worked on.

**Comment by kahgoh** [comment]
I'd like to take this one on (following on from #1934).
